lib/improver/tests/utilities/test_GenerateTimeLaggedEnsemble.py

```diff
@@ -199,6 +201,41 @@ def test_single_cube(self):
         result = GenerateTimeLaggedEnsemble().process(input_cubelist)
         self.assertEqual(result, expected_cube)
 
+    def test_non_monotonic_realizations(self):
+        """Test handling of case where realization coordinates cannot be
+        concatenated into a monotonic coordinate (eg 09Z cycle from
+        MOGREPS-UK)"""
+        data = 275.*np.ones((3, 3, 3), dtype=np.float32)
+        cycletime = dt(2019, 6, 24, 9)
+        cube1 = set_up_variable_cube(
```

Comment:
If preferred, this test could be simplified by only including cube5 and cube6, as this would still represent the key behaviour for a non-monotonic coordinate, although it depends whether it is thought that this explicit representation of the scenario is useful.

Reply:
Good point.  I'll make it a bit more concise.